document/core/exec/instructions.rst

```diff
@@ -891,35 +896,31 @@ SIMD instructions are defined in terms of generic numeric operators applied lane
 8. Push the value :math:`\V128.\VCONST~c` onto the stack.
```

Comment:
This is probably an earlier typo, should be 7.

Reply:
Fixed.